obal/data/modules/rpmspec_changelog_check.py

```diff
@@ -22,8 +44,12 @@ def run_module():
 
     result = dict(
         changed=False,
-        changelog_version_release='',
-        specfile_version_release=''
+        changelog=dict(
```

Comment:
I'm not sure this change is really needed and adds a lot of readability. I think it mostly makes it longer.